docs/CHANGELOG.md

```diff
@@ -48,6 +48,13 @@ Help: "The total number of eth node connection errors",
 
 - Support for arbitrum
 
+- Experimental: Add `DATABASE_BACKUP_ENABLED` and `DATABASE_BACKUP_FREQUENCY` configuration variables 
```

Comment:
Better mention optional URL as well, and why a node op might want to use it (to avoid excessive DB load)

Reply:
ok